src/ProgressReportable.php

```diff
@@ -19,6 +19,6 @@ public function withProgressReport(ProgressBar $progressBar): void
 
     private function progressBar(): ProgressBar
     {
-        return $this->progressBar ?: new ProgressBar(new NullOutput());
+        return $this->progressBar;
```

Comment:
This fix breaks some tests

Reply:
This line shouldn't be removed. I think the type of property should be specified as `ProgressBar|null`